cpp/models/abm/parameters.h

```diff
@@ -245,7 +245,7 @@ struct AerosolTransmissionRates {
     using Type = CustomIndexArray<ScalarType, VirusVariant>;
     static Type get_default(AgeGroup /*size*/)
     {
-        return Type({VirusVariant::Count}, 1.0); // amount of infections per m^3 per day
```

Comment:
Maybe add this comment on top of the struct AerosolTransmissionRates.

Reply:
I just thought it isn't describtive enough. What does amount if infections per m^3 even mean? It heavily depends on the amount of peopple in a room etc.

Reply:
Ok. I also think the name is clear enough without additional comment.